src/praisonai/praisonai/cli/features/gateway.py

```
@@ -244,6 +244,105 @@ def _force_kill_process(self, pid: int) -> None:
+        """
+        import yaml
+
+        action = getattr(args, "hooks_command", None)
+        config_path = getattr(args, "config_file", None) or "gateway.yaml"
+
+        def _load() -> Dict:
+            if os.path.exists(config_path):
+                try:
+                    with open(config_path, "r") as f:
+                        return yaml.safe_load(f) or {}
+                except Exception as e:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)